WebAPI/Data/Programmability/Stored Procedures/spOrders_RemoveOrderItem.sql

```diff
@@ -0,0 +1,34 @@
+﻿USE [ProductDB]
+GO
+
+------------------------------------------------------------------------------
+IF EXISTS(
+    SELECT *
+    FROM INFORMATION_SCHEMA.ROUTINES
+    WHERE [ROUTINE_NAME] = 'spOrders_RemoveOrderItem'
+        AND [ROUTINE_TYPE] = 'PROCEDURE'
+        AND [ROUTINE_BODY] = 'SQL'
+        AND [SPECIFIC_SCHEMA] = 'dbo')
+    BEGIN
+        DROP PROCEDURE dbo.spOrders_RemoveOrderItem;
+    END
+GO
+------------------------------------------------------------------------------
+
```

Comment:
Add validation.

IF NOT EXISTS (SELECT 1 FROM [dbo].[User] AS users WITH(NOLOCK) WHERE users.[UserId] = @UserId)
BEGIN
    RAISERROR ('Must pass an existing user via @UserId parameter.', 11, 2);
    RETURN -1;
END